root/edit/details/AddWork.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import commaOnlyList from '../../static/scripts/common/i18n/commaOnlyList';
+
+
+ type AddWorkEditProps = {
+   ...EditT,
+   +display_data: {
+     +attributes?: AttributeT,
+     +comment?: string,
+     +iswc?: string,
+     +language?: LanguageT,
+     +languages: $ReadOnlyArray<string>,
+     +name: string,
+     +type?: WorkTypeT,
+     +work: WorkT,
+   },
+ };
+
+type AttributeT = {
+  [key: string]: $ReadOnlyArray<WorkAttributeT>,
+  };
+
+
+const AddWork = ({edit}: {edit: AddWorkEditProps}) => {
+  const display = edit.display_data;
+  const attributes = display.attributes ?? {};
+  const type = display.type;
+  const language = display.language;
+  return (
+    <>
+      <table className="details">
+        <tr>
+          <th>{addColonText(l('Work'))}</th>
+          <td><EntityLink entity={display.work} /></td>
+        </tr>
+      </table>
+      <table className="details add-work">
+        <tr>
+          <th>{addColonText(l('Name'))}</th>
+          <td>{display.name}</td>
+        </tr>
+        {display.comment ? (
+          <tr>
+            <th>{addColonText(l('Disambiguation'))}</th>
+            <td>{display.comment}</td>
+          </tr>
+        ) : null}
+        {display.iswc ? (
+          <tr>
+            <th>{addColonText(l('ISWC'))}</th>
+            <td>{display.iswc}</td>
+          </tr>
+        ) : null}
+        {type ? (
+          <tr>
+            <th>{addColonText(l('Type'))}</th>
+            <td>{lp_attributes(type.name, 'work_type')}</td>
+          </tr>
+        ) : null}
+        {language ? (
+          <tr>
+            <th>{addColonText(l('Language'))}</th>
+            <td>{language.name}</td>
+          </tr>
+        ) : null}
+        {display.languages.length ? (
+          <tr>
+            <th>{addColonText(l('Lyrics Languages'))}</th>
+            <td>
+              {commaOnlyList(display.languages.map(
+                language => l_languages(language),
+              ))}
+            </td>
+          </tr>
+        ) : null}
+        {Object.keys(attributes).sort().map((attributeType) => {
+          return (
+            <tr key={attributeType}>
```

Comment:
You can use the short form ` => (` above and then remove the `return` here. That removes a couple lines and a level of indentation.